cli/ballerina-packerina/src/main/java/org/ballerinalang/packerina/cmd/BuildCommand.java

```diff
@@ -215,7 +215,7 @@ public void execute() {
                 Path findRoot = ProjectDirs.findProjectRoot(this.sourceRootPath);
                 if (null == findRoot) {
                     CommandUtil.printError(this.errStream,
-                            "you are trying to build/compile a ballerina project but there is no Ballerina.toml file.",
+                            "need a Ballerina.toml file to build/compile a Ballerina project.",
```

Comment:
How about something like 
```suggestion
                            "a Ballerina.toml file is required to build/compile a Ballerina project.",
```

Reply:
Fixed.